modules/core/shared/src/main/scala/gem/enum/KeywordName.scala

```diff
@@ -668,10 +668,45 @@ object KeywordName {
QTWN32, REQTWP32, REQTWS33, REQTWD33, REQTWN33, REQTWP33, REQTWS34, REQTWD34, REQTWN34, REQTWP34, REQTWS35, REQTWD35, REQTWN35, REQTWP35, REQTWS36, REQTWD36, REQTWN36, REQTWP36, REQTWS37, REQTWD37, REQTWN37, REQTWP37, REQTWS38, REQTWD38, REQTWN38, REQTWP38, REQTWS39, REQTWD39, REQTWN39, REQTWP39, REQTWS40, REQTWD40, REQTWN40, REQTWP40, REQTWS41, REQTWD41, REQTWN41, REQTWP41, REQTWS42, REQTWD42, REQTWN42, REQTWP42, REQTWS43, REQTWD43, REQTWN43, REQTWP43, REQTWS44, REQTWD44, REQTWN44, REQTWP44, REQTWS45, REQTWD45, REQTWN45, REQTWP45, REQTWS46, REQTWD46, REQTWN46, REQTWP46, REQTWS47, REQTWD47, REQTWN47, REQTWP47, REQTWS48, REQTWD48, REQTWN48, REQTWP48, REQTWS49, REQTWD49, REQTWN49, REQTWP49, REQTWS50, REQTWD50, REQTWN50, REQTWP50, REQTWS51, REQTWD51, REQTWN51, REQTWP51, REQTWS52, REQTWD52, REQTWN52, REQTWP52, REQTWS53, REQTWD53, REQTWN53, REQTWP53, REQTWS54, REQTWD54, REQTWN54, REQTWP54, REQTWS55, REQTWD55, REQTWN55, REQTWP55, REQTWS56, REQTWD56, REQTWN56, REQTWP56, REQTWS57, REQTWD57, REQTWN57, REQTWP57, REQTWS58, REQTWD58, REQTWN58, REQTWP58, REQTWS59, REQTWD59, REQTWN59, REQTWP59, REQTWS60, REQTWD60, REQTWN60, REQTWP60, REQTWS61, REQTWD61, REQTWN61, REQTWP61, REQTWS62, REQTWD62, REQTWN62, REQTWP62, REQTWS63, REQTWD63, REQTWN63, REQTWP63, REQTWS64, REQTWD64, REQTWN64, REQTWP64, REQTWS65, REQTWD65, REQTWN65, REQTWP65, REQTWS66, REQTWD66, REQTWN66, REQTWP66, REQTWS67, REQTWD67, REQTWN67, REQTWP67, REQTWS68, REQTWD68, REQTWN68, REQTWP68, REQTWS69, REQTWD69, REQTWN69, REQTWP69, REQTWS70, REQTWD70, REQTWN70, REQTWP70, REQTWS71, REQTWD71, REQTWN71, REQTWP71, REQTWS72, REQTWD72, REQTWN72, REQTWP72, REQTWS73, REQTWD73, REQTWN73, REQTWP73, REQTWS74, REQTWD74, REQTWN74, REQTWP74, REQTWS75, REQTWD75, REQTWN75, REQTWP75, REQTWS76, REQTWD76, REQTWN76, REQTWP76, REQTWS77, REQTWD77, REQTWN77, REQTWP77, REQTWS78, REQTWD78, REQTWN78, REQTWP78, REQTWS79, REQTWD79, REQTWN79, REQTWP79, REQTWS80, REQTWD80, REQTWN80, REQTWP80, REQTWS81, REQTWD81, REQTWN81, REQTWP81, REQTWS82, REQTWD82, REQTWN82, REQTWP82, REQTWS83, REQTWD83, REQTWN83, REQTWP83, REQTWS84, REQTWD84, REQTWN84, REQTWP84, REQTWS85, REQTWD85, REQTWN85, REQTWP85, REQTWS86, REQTWD86, REQTWN86, REQTWP86, REQTWS87, REQTWD87, REQTWN87, REQTWP87, REQTWS88, REQTWD88, REQTWN88, REQTWP88, REQTWS89, REQTWD89, REQTWN89, REQTWP89, REQTWS90, REQTWD90, REQTWN90, REQTWP90, REQTWS91, REQTWD91, REQTWN91, REQTWP91, REQTWS92, REQTWD92, REQTWN92, REQTWP92, REQTWS93, REQTWD93, REQTWN93, REQTWP93, REQTWS94, REQTWD94, REQTWN94, REQTWP94, REQTWS95, REQTWD95, REQTWN95, REQTWP95, REQTWS96, REQTWD96, REQTWN96, REQTWP96, REQTWS97, REQTWD97, REQTWN97, REQTWP97, REQTWS98, REQTWD98, REQTWN98, REQTWP98, REQTWS99, REQTWD99, REQTWN99, REQTWP99, COADDS, EXPTIME, FILTER3, FOCUSNAM, FOCUSPOS, FPMASK, BEAMSPLT, WINDCOVR, FRMSPCYCL, HDRTIMING, LNRS, MODE, NDAVGS, PVIEW, TDETABS, TIME, TMOUNT, UCODENAM, UCODETYP, VDDCL1, VDDCL2, VDDUC, VDET, VGGCL1, VGGCL2, VSET, A_TDETABS, A_TMOUNT, A_VDDCL1, A_VDDCL2, A_VDDUC, A_VDET, A_VGGCL1, A_VGGCL2, A_VSET)
```

Comment:
There are so many of these.  Does it make sense to mix instruments, if that is what is going on?

Reply:
I wouldn't worry too much about this list growing large, because it is generated code. I care more about the keeping things readable and tidy in the sql files.